src/BTVNanoCommissioning/utils/AK4_parameters.py

```diff
@@ -125,5 +150,7 @@
             "MC": "calibeHistoWrite_MC2023_Summer23BPix.root",
         },
     },
-    "CAMPAIGN_prompt_dataMC": {"lumiMask": "$LUMIMASK_DATAMC"},
```

Comment:
Could you please reverse this back to the `"CAMPAIGN_prompt_dataMC": {"lumiMask": "$LUMIMASK_DATAMC"}` since this should be modified later in the submission script

Reply:
Reversed the change